osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/application/config/DomainAdapterInitializer.java

```
@@ -44,11 +45,17 @@ public void onStartup(final ServletContext servletContext) throws ServletExcepti
             initialContext = new InitialContext();
             final String logLocation = (String) initialContext
                     .lookup("java:comp/env/osgp/AdapterDomainAdmin/log-config");
-            LogbackConfigurer.initLogging(logLocation);
+            
+            if ((new File(logLocation).exists())) {
+                LogbackConfigurer.initLogging(logLocation);
+            } else {
+                LogbackConfigurer.initLogging("classpath:logback.xml");
+            }
         } catch (final NameNotFoundException | FileNotFoundException | JoranException e) {
             // Do nothing, if the file referred in context.xml is not found,
             // the default logback.xml will be used.
-            LOGGER.info("Using classpath logback.xml");
+            LOGGER.info("Caught an exception [" + e.getMessage() + "]");
```

This will not stop Jenkins/SonarQube from complaining the exception is not handled propertly I think.
Was there any objection to changing the following line to something like:
`LOGGER.info("Using classpath logback.xml.", e);`